staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/metrics/metrics.go

```diff
@@ -46,9 +56,13 @@ var (
 	lockLastFromStorage sync.Mutex
 	lockLastToStorage   sync.Mutex
 
-	lastFromStorage time.Time
-	lastToStorage   time.Time
+	lastFromStorage                                 time.Time
+	lastToStorage                                   time.Time
+	keyIDHashTotalMetricLabels                      *lru.Cache
+	keyIDHashStatusLastTimestampSecondsMetricLabels *lru.Cache
+	cacheSize                                       int = 10
```

Comment:
```suggestion
	cacheSize                                       = 10
```

Reply:
This is in var it expects a type or we will get `syntax error: unexpected literal 10, expected type`